test/pkg/environment/aws/expectations.go

```diff
@@ -214,3 +215,18 @@ func (env *Environment) GetCustomAMI(amiPath string, versionOffset int) string {
 	Expect(err).To(BeNil())
 	return *parameter.Parameter.Value
 }
+
+func (env *Environment) GetNthLastAMIVersion(k8sVersion string, n int) string {
```

Comment:
Is it possible to extend this off of the `GetCustomAMI` function rather than creating an entirely different function or to create a function similar to it so that it makes sense that the functions are related?

Reply:
Will implement in a follow up PR